deploy/AzureAppServices/Datadog.Trace.AzureAppServices.nuspec

```
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>Datadog.Trace.AzureAppServices</id>
+    <version>1.10.0</version>
+    <title>Datadog APM - .NET Tracer</title>
+    <authors>Robert Glickman;Lucas Pimentel-Ordyna;Colin Higgins</authors>
```

Keep this in sync with the other NuGet packages?
```suggestion
    <authors>lucas.pimentel.datadog;colinhigginsdatadog;zachmontoyadd;bobuva</authors>
```